Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -674,6 +672,34 @@ private void LoadHitObjects(IList<List<Texture2D>> hitObjects, string element, i
         /// <returns></returns>
         private string GetResourcePath(string element) => $"{element}";
 
+        /// <summary>
+        ///     Loads hit and holdlights
+        /// </summary>
+        /// <param name="lightingOjects"></param>
+        /// <param name="type">Either hitlighting or holdlighting</param>
+        /// <param name="lane"></param>
+        /// <returns></returns>
+        private void LoadLighting(List<List<Texture2D>> lightingObjects, string type, int lane)
+        {
+            var mode = ModeHelper.ToShortHand(Mode).ToLower();
+
+            if (!Directory.Exists($"{Store.Dir}/{mode}"))
+                return;
+            
+            var folder = $"{Store.Dir}/{mode}/Lighting";
+
+            // Check for lane-specific lighting; otherwise, load general lighting for the lane
+            if (Directory.EnumerateFiles(folder).Any(f => f.Contains($"{type}-{lane}")))
```

Comment:
Naming a file to something like hitlighting-2abc.png would cause this to try to loadhitlighting-2.png even though it does not exist.

Reply:
Not sure how I missed that, updated the code to use Regex.IsMatch to guarantee the correct file is searched for.